db/migrate/20180807090734_remove_place_cord_from_event.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePlaceCordFromEvent < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :events, :place_cord, :string
```

Comment:
In the context of the task, you could do the following:
1) Rollback migrations
2) Create table without place_cord column.

Thanks to that we will merge the PR without additional "garbage" migration

Reply:
See `rails db:rollback`, `rails db:migrate:status`